providers/dns/servercow/internal/client_test.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func setupAPIMock() (*Client, *http.ServeMux, func()) {
 	handler := http.NewServeMux()
-	svr := httptest.NewServer(handler)
+	svr := httptest.NewServer(handler) // FIME t.Cleanup(server.Close)
```

Comment:
Is this FIXME intentional?

Reply:
oops!

Reply:
if you like `t.Cleanup`, you can take a look to `t.Setenv()`

Reply:
Neat!